### DISCUSSION TITLE
Update Fails: "Fatal: not in a git respository"

### DISCUSSION THREAD

**Discussion by kevrenac** [ohmyzsh/ohmyzsh#9815] [Help] [answered]
Hi all,

OMZ auto-update just ran on opening my terminal, and I got a bunch of "fatal: not in a git repository" messages. Running the manual update command fails as well. No joy either on restarting the terminal or my computer. Any ideas on how to fix this? I'd prefer not to re-install from scratch.

**Comment by mcornella** [comment, marked as answer]
Did you perhaps install Oh My Zsh via downloading a zip file of the repository, instead of using the install script? Or maybe you deleted the `.git` directory inside of `$ZSH`? If that's the case, you'll have to reinstall it. You can do that without using the install script by deleting `$ZSH` and running `git clone https://github.com/ohmyzsh/ohmyzsh.git <previous-$ZSH-location>`.

**Reply by kevrenac** [reply]
Yep, the .git directory was deleted, not sure how I managed to do that. Reinstalling fixed it. Thanks!
